### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Arthur Jacobs

### DISCUSSION THREAD

**Discussion by Arthur2479** [heig-vd-dai-course/heig-vd-dai-course#262] [Show and tell]
https://github.com/Arthur2479

**Comment by ludelafo** [comment]
Very nice profile, thank you!

What kind of photography do you do? :)

**Reply by Arthur2479** [reply]
Thanks!
Mostly sport / party photography, but I like to try everything ;)

**Reply by ludelafo** [reply]
Cool! Maybe you'll find a specific use for the first practical work to process your pictures starting next week! :)
